eras/shelley-ma/impl/src/Cardano/Ledger/ShelleyMA/TxBody.hs

```diff
@@ -287,19 +287,19 @@ pattern TxBody' ::
   StrictMaybe (AuxiliaryDataHash (EraCrypto era)) ->
   MultiAsset (EraCrypto era) ->
   MATxBody era
-pattern TxBody' {inputs', outputs', certs', wdrls', txfee', vldt', update', adHash', mint'} <-
+pattern TxBody' {smatbrInputs', smatbrOutputs', smatbrCerts', smatbrWdrls', smatbrTxFee', smatbrValidityInterval', smatbrTxUpdate', smatbrAuxDataHash', smatbrMint'} <-
```

Comment:
These actually don't need the `r` at the end, since they don't operate on the `Raw` type, but on the . Also removing the ugly tick is a good idea:
```suggestion
pattern TxBody' {matbInputs, matbOutputs, matbCerts, matbWdrls, matbTxFee, matbValidityInterval, matrTxUpdate, matbAuxDataHash, matbMint} <-
```

Reply:
Thanks for the suggestion :+1: I also renamed `TxBody'` to `MATxBody_`